godot-core/src/builtin/callable.rs

```diff
@@ -16,6 +16,12 @@ use crate::obj::{Gd, GodotClass, InstanceId};
 use std::{fmt, ptr};
 use sys::{ffi_methods, GodotFfi};
 
+#[cfg(before_api = "4.3")]
+#[cfg(since_api = "4.2")]
+type GDExtensionCallableCustomInfo = sys::GDExtensionCallableCustomInfo;
```

Comment:
You can use `all(...)` here :slightly_smiling_face: 
```suggestion
#[cfg(all(since_api = "4.2", before_api = "4.3"))]
type GDExtensionCallableCustomInfo = sys::GDExtensionCallableCustomInfo;
```